main/CommandHandler.cpp

```diff
@@ -1618,6 +1619,47 @@ int downloadOTA(const uint8_t command[], uint8_t response[])
   return 6;
 }
 
+int brSetECTrustAnchor(const uint8_t command[], uint8_t response[])
+{
```

Comment:
Can you add a description of how yo-u expect command to appear? like done here https://github.com/arduino/nina-fw/blob/331d584a1ac8a74f6fdd1d24a9e7e7d085c65355/main/CommandHandler.cpp#L1624-L1630

This will help in the future to debug this command handle.